live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/adapters/messaging/in/mapper/property/PropertiesMapper.java

```diff
@@ -77,6 +73,49 @@ public Set<CustomPropertyDelta<?>> calculateCustomPropertiesDelta(RepoEvent<Data
         return someCustomPropertiesUpdated(event);
     }
 
+    private Set<CustomPropertyDelta<?>> allCustomPropertiesUpdated(RepoEvent<DataAttributes<NodeResource>> event)
+    {
+        Stream<CustomPropertyDelta<?>> properties = customPropertiesStream(event.getData().getResource())
+                .filter(property -> Objects.nonNull(property.getValue()))
+                .map(property -> CustomPropertyDelta.updated(property.getKey(), property.getValue()));
+
+        Stream<CustomPropertyDelta<?>> nameProperty = calculateNamePropertyDelta(event);
+
+        return Stream.concat(properties, nameProperty)
+                .collect(Collectors.toSet());
+    }
+
+    private Set<CustomPropertyDelta<?>> someCustomPropertiesUpdated(RepoEvent<DataAttributes<NodeResource>> event)
+    {
+        return Stream.concat(
+                calculatePropertiesDelta(event),
+                calculateNamePropertyDelta(event)).collect(Collectors.toSet());
+    }
+
+    private Stream<CustomPropertyDelta<?>> calculatePropertiesDelta(RepoEvent<DataAttributes<NodeResource>> event)
+    {
+        if (isFieldUnchanged(event, NodeResource::getProperties))
+        {
+            return Stream.empty();
+        }
+
+        return customPropertiesStream(event.getData().getResourceBefore())
+                .map(oldPropertyEntry -> toCustomPropertyDelta(event.getData(), oldPropertyEntry));
+    }
+
+    private Stream<CustomPropertyDelta<?>> calculateNamePropertyDelta(RepoEvent<DataAttributes<NodeResource>> event)
+    {
+        if (shouldNotUpdateField(event, NodeResource::getName))
+        {
+            return Stream.empty();
+        }
+
+        return ofNullable(event.getData().getResource().getName())
+                .stream()
+                .filter(Objects::nonNull)
+                .map(name -> CustomPropertyDelta.updated(NAME_PROPERTY_KEY, name));
```

Comment:
should we use "fully qualified name" or simple name or maybe both? wdyt? I would say we should use just fully qualified one and ignore simple name

Reply:
How does it looks in case of other properties? Do we use the "fully qualified name"?

Reply:
so for other "custom" properties keys we use fully qualified names (as it comes in event), but idk if any other property can have two different "values", I'll maybe show an example:

we would have
`cm:title` not `title` property key and its value can be `some interesting title`
for name we would have
`cm:name` not `name` as property key but as value we could have both `cm:my_file_name` and `my_flie_name`, and the question is, whether we want/need to store both or whether one is enough (and in that case which one we prefer)

Reply:
I would stick with just `cm:name` as the only name. I asked a very similar question to the Hx Insight team and they weren't interested in having any "well known" field names other than an id.

Reply:
The most safe approach would be to preserve both values as I'm note sure if HxI will require them both or one of them 🤔 Maybe @tpage-alfresco can tell something more in this topic

Reply:
what about values? `cm:my_file_name`/ `my_file_name`/ both?

Reply:
Pawel clarified that `cm:my_file_name` is coming from `primaryAssocQName`. This field doesn't give any additional useful information that isn't already present in other fields. I would stick with just `cm:name` and so the value should just be `my_file_name` without a prefix.